dhlab/api/dhlab_api.py

```diff
@@ -63,13 +82,15 @@ def ner_from_urn(
 
 
 def pos_from_urn(
-    urn: str | None = None, model: str | None = None, start_page=0, to_page=0
+    urn: str | None = None, model: str | None = None, start_page: int | None = 0, to_page: int | None = 0
```

Comment:
Same as above, the default value prevents the parameters from being `None`

```suggestion
    urn: str | None = None, model: str | None = None, start_page: int = 0, to_page: int = 0
```